### ISSUE TITLE
Using IdentityServer to manage a users access permission to other websites

### ISSUE THREAD

**Issue by earnshaw** [IdentityServer/IdentityServer3#2648]
I am new to this and to be honest it's a bit mind boggling - there is just too much to know coming straight into this cold.

Even doing some of the tutorials, don't really help me that much as I don't really know whats going on other than a token is being passed about and it has a time limit on it.

At the moment I'm getting totally bogged down with Roles, Scopes, Claims, API's and Users vs Clients - they all seem to sound like they do similar types of jobs.

Anyway....what is the best way to learn how a user can login and have access to various websites you have published.  Or put it another way the website should check if the logged in user has valid access to it's services.  I suspect a tutorial, but the chances of finding a tutorial that does exactly what you need is unlikely!

What I want to do is: you login to a landing page (or a portal as I call it) - and it shows you a list of websites you have access to and greys out ones you don't, they could be text or icons (doesn't matter).  When you click on one you have access to, it takes you to that website (along with your valid token?? I think), you leave the portal behind, and the target website does a check as well to make sure that token is valid for it's own website and that you can continue on using it's particular services.  I'm expecting a bit of chat here between the website and IdentityServer.

Anyway, if anyone can help in plain English that would be great! :)

**Comment by leastprivilege** [comment]
This forum is really for answering specific questions - I think what you need is consulting/mentoring. Let us know if we can help.

**Comment by earnshaw** [comment]
ok - feel free to delete this - so is it just stackoverflow or something like that - which would be better for asking questions like this.  I would like to get some consulting in - but we are freezing expenditure at the moment until the oil price picks up!

**Comment by leastprivilege** [comment]
OK. Good luck with the oil ;)
